Bottom: d9e0c2077a0f3f4c3c05b656baab44d46f0dc68f
Top:    b075b23ae879f12fa89bb798d10a8f34cbc1c8f0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 14:41:17 +0000

Refresh of wip-on-quad-edge-play

---

diff --git a/src/hid/gtk/appearance.c b/src/hid/common/appearance.c
similarity index 100%
rename from src/hid/gtk/appearance.c
rename to src/hid/common/appearance.c
diff --git a/src/hid/gtk/appearance.h b/src/hid/common/appearance.h
similarity index 100%
rename from src/hid/gtk/appearance.h
rename to src/hid/common/appearance.h
